assemblies/dynamic-plugins/assembly-third-party-plugins-installation.adoc

```diff
@@ -0,0 +1,114 @@
+
+To integrate a third-party plugin into {product-short}:
+
+. First, obtain the plugin's source code.
+. Export the plugin as a dynamic plugin package. See xref:proc-export-third-party-plugins-rhdh_{context}[].
+. Package and publish the dynamic plugin. See xref:proc-package-third-party-dynamic-plugin_{context}[].
+. Install the plugin in the {product-short} environment. See xref:proc-install-third-party-plugins-rhdh_{context}[].
+
+The third-party plugins include:
+
+Backend plugins::
++
+--
+To ensure compatibility with the dynamic plugin support and enable their use as dynamic plugins, existing backend plugins must be compatible with, the new backend system. Additionally, these plugins must be rebuilt using a dedicated CLI command.
+
+The new backend system is created using `createBackendPlugin()` or `createBackendModule()`. You must export the new backend system as the default export from either the main package or an alpha package (if support exists in alpha). The dynamic export mechanism automatically identifies private dependencies, and adds them to the `bundleDependencies` field in the `package.json` file. This mechanism ensures that the dynamic plugin package is self-contained, with private dependencies bundled within a private `node_modules` folder.
+
+The backend plugins contains two types of dependencies, including:
+
+* *Shared dependencies* are provided by an application and listed as `peerDependencies` in `package.json` file, not bundled in the dynamic plugin package. For example, by default, all `@backstage` packages are shared.
++
+You can use the `--shared-package` flag to specify shared dependencies. To treat a `@backstage` package as private, use the negation prefix (`!`).
+
+* *Embedded dependencies* are bundled into the dynamic plugin package with their dependencies hoisted to the top level. By default, packages with `-node` or `-common` suffixes are embedded.
++
+You can use the `--embed-package` flag to specify additional embedded packages, including those in the same repository that do not follow the default naming convention.
+
+The following is an example of exporting a dynamic plugin with shared and embedded packages:
+
+.Example dynamic plugin export with shared and embedded packages
+[source,bash]
+----
+npx @janus-idp/cli@latest export-dynamic-plugin --shared-package '!/@backstage/plugin-notifications/'<1> --embed-package @backstage/plugin-notifications-backend <2>
+----
+
+<1> `@backstage/plugin-notifications` package is treated as a private dependency and is bundled in the dynamic plugin package, despite being in the `@backstage` scope.
+<2> `@backstage/plugin-notifications-backend` package is marked as an embedded dependency and is bundled in the dynamic plugin package.
+--
+
+Front-end plugins::
++
+--
+Front-end plugins can use `scalprum` for configuration, which the CLI can generate automatically during the export process. The generated default configuration is logged when running the following command:
```

Comment:
Shouldn't this section be in the document about exporting dynamic plugins?  It's not clear to me how this relates to installing third party plugins.

Reply:
Moved this content to export section